File: SWIFT/day12.swift
class Grade : Student{
   private score : Int;
    
   init(fname : String,lname : String,p : Int,s : Int){
         super.init(fname,lname: lname,p: p)
         score=s
   }
    
   func calculate () -> Character{
     switch score {
     case where score<40 :
        return 'D'
     case where score<60 :
        retutn 'B'
     case where score<75 :
        retutn 'A'
     case where score<90 :
        retutn 'E'
     default :
        retutn '0'
     }
   }
   
}

File: PYTHON3/day12.py
class Grade(Student):
    def __init__(self,fn, ln, p, score):
        super().__init__(fn, ln, p)
        self.score = score
    def calculate(self):
        if self.score < 40:
            return "D"
        elif self.score >= 40 and self.score <60:
            return "B"
        elif self.score >= 60 and self.score<75:
            return "A"
        elif self.score >=75 and self.score <90:
            return "E"
        elif self.score >=90 and self.score <=100:
            return "O"
        
   